src/vdaf.rs

```diff
@@ -163,64 +164,25 @@ pub trait Aggregator: Vdaf {
         input_share: &Self::InputShare,
     ) -> Result<Self::PrepareStep, VdafError>;
 
-    /// Preprocess a round of messages into a single input to [`Aggregator::prepare_step`].
+    /// Preprocess a round of prepare messages into a single input to [`Aggregator::prepare_step`].
     fn prepare_preprocess<M: IntoIterator<Item = Self::PrepareMessage>>(
         &self,
         inputs: M,
     ) -> Result<Self::PrepareMessage, VdafError>;
 
     /// Compute the next state transition from the current state and the previous round of input
-    /// messages.
+    /// messages. If this returns [`PrepareTransition::Continue`], then the returned
+    /// [`Self::PrepareMessage`] should be combined with the other aggregator's `PrepareMessage`s from
+    /// this round and passed into another call to this method. This continues until this method
+    /// returns [`PrepareTransition::Finish`], at which point the returned output share may be
+    /// aggregated. If the method returns [`PrepareTransition::Fail`], the aggregator should alert
+    /// its peers as specified in the PPM protocol.
```

Comment:
It's best to not be too specific about the application.

```suggestion
    /// aggregated. If the method returns [`PrepareTransition::Fail`], the aggregator should consider its input share invalid and not attempt to process it any further.
```